frontend/src/components/SocialMediaLink/index.js

```diff
@@ -0,0 +1,33 @@
+import "./index.css";
+
+import Default from "../../assets/imgs/socialButtons/default.png";
+import Github from "../../assets/imgs/socialButtons/github.png";
+import Linkedin from "../../assets/imgs/socialButtons/linkedin.png";
+import Behance from "../../assets/imgs/socialButtons/behance.png";
+import Facebook from "../../assets/imgs/socialButtons/facebook.png";
+import Instagram from "../../assets/imgs/socialButtons/instagram.png";
+import Dribbble from "../../assets/imgs/socialButtons/dribble.png";
+import Medium from "../../assets/imgs/socialButtons/medium.png";
+import Twitter from "../../assets/imgs/socialButtons/twitter.png";
+
+const socialMediaLink = ({ name, username }) => {
+  const info = {
+    "": { url: "#", logo: Default },
+    github: { url: `https://github.com/${username}`, logo: Github },
+    linkedin: { url: `https://www.linkedin.com/company/${username}`, logo: Linkedin },
+    behance: { url: `https://www.behance.net/${username}`, logo: Behance },
+    facebook: { url: `https://www.facebook.com/${username}`, logo: Facebook },
+    instagram: { url: `https://www.instagram.com/${username}`, logo: Instagram },
+    dribbble: { url: `https://dribbble.com/${username}`, logo: Dribbble },
+    medium: { url: `https://medium.com/${username}`, logo: Medium },
+    twitter: { url: `https://twitter.com/${username}`, logo: Twitter },
+  }
```

Comment:
I'm not sure this data should "live" in this component either. We could think of extracting it into a `constants.js` file. Then you could import the data you need in the parent components, and pass the props down. What do you think @joaojusto ?

Reply:
@joaojusto Nevermind. They're planning to move all of this to the database.